components/formats-gpl/src/loci/formats/in/CV7000Reader.java

```diff
@@ -309,6 +309,10 @@ public int compare(Channel c1, Channel c2) {
 
     for (Plane p : planeData) {
       if (p != null) {
+        if (!isWellAcquired(p.field.row, p.field.column)) {
```

Comment:
Overall, the logic is to populate a more faithful `uniqueWells` (excluding the missing wells) and then initialize and populate a `reversePlaneLookup` map only for the existing wells which makes total sense.

My primary concern with the proposed fix is its computational cost and the potential performance impact on large plates. `isWellAcquired` returns the existence of a particular well from a loop on `PlaneData` itself. Having this call nested within a `PlaneData` loop means an increase of complexity which will scale with the number of wells/fields in a plate.

Reply:
Agreed. I wasn't really thinking about performance for the initial fix, was really trying to verify that it worked. 8e1dac2a95 is one option that should make `isWellAcquired` inexpensive for repeated checks on the same well.